docs/src/pages/components/data-grid/accessibility/accessibility.md

```diff
@@ -71,6 +71,7 @@ Use the arrow keys to move the focus.
 |                                             <kbd class="key">CTRL</kbd> + <kbd class="key">A</kbd> | Select all rows                                   |
 |                                             <kbd class="key">CTRL</kbd> + <kbd class="key">C</kbd> | Copy the currently selected row                   |
 |                                                        <kbd class="key">CTRL</kbd> + Click on cell | Enable multi-selection                            |
```

Comment:
**Off-topic**

It looks like we can do a follow-up to fix the grammar.

Reply:
I don't see an issue with the grammar, but is the description correct?

Reply:
Ok, I was wondering about the plural, then: https://github.com/mui-org/material-ui-x/pull/1813/files#r644343155.